pkgs/development/node-packages/node-packages-v4.nix

```diff
@@ -73,7 +73,7 @@ let
       version = "3.0.4";
       src = fetchurl {
         url = "https://registry.npmjs.org/minimatch/-/minimatch-3.0.4.tgz";
-        sha512 = "1879a3j85h92ypvb7lpv1dqpcxl49rqnbgs5la18zmj1yqhwl60c2m74254wbr5pp3znckqpkg9dvjyrz6hfz8b9vag5a3j910db4f8";
+        sha1 = "5166e286457f03306064be5497e8dbb0c3d32083";
```

Comment:
The new version of node2nix now uses the more secure `sha512`. I rerun the script to solve the merge conflict.

Reply:
Thank you. I didn't have path to this repo in my $NIX_PATH, that is the reason why `generate.sh` use `node2nix` version 1.2.0